### PR TITLE
Pull request demo

### PR DESCRIPTION
**Pull request by cdosoriob**

### CODE REVIEW

**Comment by cdosoriob** [comment]
First comment from cdosoriob

File: Queries/PlutoContext.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public PlutoContext()
             this.Configuration.LazyLoadingEnabled = false;
 
             //cosoriob edit
+            //cosoriob edit en branch para_PR
```
Comment (cdosoriob) [inline Queries/PlutoContext.cs:16]: example of comments on particular lines

Comment (cdosorio) [reply Queries/PlutoContext.cs:16]: I think its ok!

**Comment by cdosorio** [comment]
First comment of owner on pull request demo    :smile:

**Comment by cdosorio** [comment]
Closing pull request demo  :relaxed:

**Comment by cdosorio** [comment]
Reopening PR  :smirk:

**Comment by cdosorio** [comment]
Reopening PR  :sunglasses:

**Pull request merged**